### ISSUE TITLE
Cloudflare

### ISSUE THREAD

**Issue by PolishLithuanianCommonwealth** [Theblockbuster1/disboard-auto-bump#32]
I'm unsure if you're still working on this; but I'll put this here.

Ever since Cloudflare's security has gone on crack, this barely works. Every-time it refreshes, and I mean EVERY TIME, Cloudflare asks to verify you.

This is virtually unusable due to this. I'm unsure if a fix can even happen. I just wanted to make you aware.

**Comment by Theblockbuster1** [comment]
You may be right, I think that the only way to fix it would be by using a paid captcha service, which would make the extension cost money to use, so I'm not sure that I would want to implement that.

[This extension](https://chrome.google.com/webstore/detail/privacy-pass/ajhmfdgkijocedmfjonnpjfojldioehi) used to work, I think a new version is currently being developed. If it's ever finished, it could be a workaround to this problem.

**Comment by PolishLithuanianCommonwealth** [comment]
Yeah, I feel the only potential workarounds could be that extension being fixed and reworked, or a possible discord bot variant being developed, but I feel that'd take a lot of time on your end, which I'd understand would be a turnoff.

**Comment by helloimaMan** [comment]
My laptop doesn't face this issue, but my Pc does. Has anyone found a fix to this cloud-flare human verification issue?

**Comment by Detrailia** [comment]
I started getting issue now
